src/html5_parser/node.rs

```diff
@@ -33,8 +34,8 @@ pub enum NodeData {
 }
 
 /// Id used to identify a node
-#[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq)]
-pub struct NodeId(usize);
+#[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq, Display)]
+pub struct NodeId(pub usize);
```

Comment:
Making the internal `usize` is probably not needed, but I think it's fine for now.  This kind of thing only matters when external libraries are depending on this code.

Reply:
I needed it for using them inside unit-tests. Maybe we can make this better by using a `NodeId::from("5")` method?

Reply:
You can say `pub(crate) usize` to make it only visible to the current crate.